buf/registry/module/v1beta1/commit_service.proto

```diff
@@ -35,10 +35,13 @@ service CommitService {
   rpc ListCommitHistory(ListCommitHistoryRequest) returns (ListCommitHistoryResponse) {
     option idempotency_level = NO_SIDE_EFFECTS;
   }
-  // Create commits on a Module with associated Content.
+  // Create commits on a Module with associated content (e.g. blobs, branches, tags).
```

Comment:
branches and tags aren't content - can you just revert this?

Reply:
```suggestion
  // Create commits on a Module with associated Content.
```